game/scripts/vscripts/heroes/skywrath_mage/skywrath_mage_arcane_bolt.lua

```diff
@@ -0,0 +1,143 @@
+-- Author: Shush
+-- Date: 2/12/2017
+
+require("heroes/skywrath_mage/skywrath_mage_target_finder")
```

Comment:
Add comment explaining what this does for this file, something like `-- This adds utility function SkywrathTargetSpellFinder()`